src/common/KeyValueTable/keyValueTable.scss

```diff
@@ -1,6 +1,6 @@
 @import '~igz-controls/scss/borders';
 @import '~igz-controls/scss/colors';
-$actionsBlockWidth: 72px;
+$actionsBlockWidth: 82px;
```

Comment:
I see we also use 82px in the DRC in common.scss. Better to have it as common variable and use it in both places